web/react/components/user_settings.jsx

```diff
@@ -13,6 +13,7 @@ var assign = require('object-assign');
 
 function getNotificationsStateFromStores() {
     var user = UserStore.getCurrentUser();
+    var soundNeeded = !utils.checkBrowserType('firefox');
```

Comment:
I think 'firefox' should be a constant here, or have a wrapper function in utils called ```isBrowserTypeFirefox()``` that can have 'firefox' hardcoded. Otherwise if someone is trying to figure out how to use ```checkBrowserType``` they don't really know what to pass in. For example, do I pass in 'FF', 'firefox', or 'Firefox' and what do I need to pass in for other browsers? Don't want to have to grep the code to find an example for something so simple